### PR TITLE
fix(v-bind-style): only change kebabCase to camelCase

### PR DESCRIPTION
**Pull request by waynzh**
fix https://github.com/vuejs/eslint-plugin-vue/issues/2409

### CODE REVIEW

File: tests/lib/rules/v-bind-style.js
Comment (FloEdelmann) [inline tests/lib/rules/v-bind-style.js:1]: Could you please add another test case `<div :foo_bar="foo_bar" />` that reports an error suggesting to fix this to `<div :foo_bar />` if the `sameNameShorthand` option is set to `always`?

**Review by FloEdelmann — COMMENTED** [review]
Thanks for the fix! I have one suggestion for an additional test case, otherwise this looks good to me already :slightly_smiling_face:

File: tests/lib/rules/v-bind-style.js
Comment (waynzh) [reply tests/lib/rules/v-bind-style.js:1]: Update ✅

**Review by FloEdelmann — APPROVED** [review]
Nice, thank you!

**Review by ota-meshi — APPROVED** [review]
LGTM! Thank you!

**Pull request merged**